src/features/prefixes.ts

```diff
@@ -0,0 +1,6 @@
+export enum Prefixes {
+  "!hellforge",
```

Comment:
This is not a good enum.
Because you did not specify the values they will be set from 0 to X accordingly.
And once compiled an enum with numbers is able to do a reverse mapping.
Meaning that the length of your enum will double.
I do not think that this is your wish and also this is risky.
So I suggest you to do:
HELLFORGE = '!hellforge', ...

Reply:
Is an interface something I can do? :o

Reply:
Nope.
If you want to create a list of something you can:
- Create an enum
- Create an array